test/unit/expiration_process_test.lua

```diff
@@ -102,3 +102,37 @@ function g.test_archive_by_timestamp(cg)
     end)
     task:kill()
 end
+
+function g.test_execution_error(cg)
```

Comment:
Yeah, the name of the test should be changed. But is_tuple_expired is not called in this test because of no data available in space in the original test. Do I need add data to correct test?

Reply:
Changed test name to `test_broken_is_tuple_expired `